docs/builds/guides/migration/migration-from-ckeditor-4.md

```diff
@@ -420,264 +420,264 @@ Note: The number of options was reduced on purpose. We understood that configuri
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-startupOutlineBlocks">startupOutlineBlocks</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-startupOutlineBlocks">startupOutlineBlocks</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-startupShowBorders">startupShowBorders</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-startupShowBorders">startupShowBorders</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesSet">stylesSet</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesSet">stylesSet</a></td>
 			<td>Not available yet. Refer to the <a href="https://github.com/ckeditor/ckeditor5/issues/5700" target="_blank" rel="noopener">relevant GitHub issue</a>.</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesheetParser_skipSelectors">stylesheetParser_skipSelectors</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesheetParser_validSelectors">stylesheetParser_validSelectors</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesheetParser_skipSelectors">stylesheetParser_skipSelectors</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-stylesheetParser_validSelectors">stylesheetParser_validSelectors</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-tabIndex">tabIndex</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-tabIndex">tabIndex</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-tabSpaces">tabSpaces</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-tabSpaces">tabSpaces</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates">templates</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_files">templates_files</a> <br> <a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_replaceContent">templates_replaceContent</a></td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates">templates</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_files">templates_files</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-templates_replaceContent">templates_replaceContent</a></td>
 			<td>N/A</td>
 		</tr>
 		<tr>
-			<td><a href="../../../../ckeditor4/latest/api/CKEDITOR_config.html#cfg-title">title</a></td>
-			<td>N/A</td>
+			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-title">title</a></td>
```

Comment:
This is not the same feature. Compare the API docs and guide of these two to check this out.

Reply:
```suggestion
			<td>N/A</td>
```